### ISSUE TITLE
IntelliJ IDEA 2016.2 Error on adding individual RuleSet

### ISSUE THREAD

**Issue by petermannel** [amitdev/PMD-Intellij#32]
Hi,
adding a Rule set ends up in Error "The selected file is not a valid PMD Ruleset: Unable to exclude rules [EmptyIfStmt, EmptyCatchBlock].
This is the Ruleset : https://github.com/wcm-io/wcm-io-tooling/blob/develop/maven/global-build-tools/src/main/resources/global-build-tools/pmd-ruleset.xml
Whats going wrong here?

**Comment by amitdev** [comment]
The rule.xml seems to have errors. The EmptyIfStmt is in empty.xml in PMD. Following is the corrected rule.xml that works:

```xml
  <rule ref="rulesets/java/empty.xml">
    <exclude name="EmptyCatchBlock"/>  <!-- false positives with empty blocks with comments as well -->
    <exclude name="EmptyIfStmt"/>
  </rule>
```

Also `BooleanInversion` rule is deprecated in PMD so remove it.
